crates/core/tedge_mapper/src/c8y_converter.rs

```diff
@@ -106,10 +116,24 @@ impl CumulocityConverter {
 
     fn try_convert_event(&mut self, input: &Message) -> Result<Vec<Message>, ConversionError> {
         let tedge_event = ThinEdgeEvent::try_from(input.topic.name.as_str(), input.payload_str()?)?;
-        let smartrest_alarm = event::serialize_event(tedge_event)?;
-        let smartrest_topic = Topic::new_unchecked(SMARTREST_PUBLISH_TOPIC);
+        dbg!(input.payload_bytes().len());
+        if input.payload_bytes().len() > MQTT_EVENT_SIZE_LIMIT {
```

Comment:
I believe there is already a check for size of the message done by `size_threshold`, do you need an explicit check here or you could reuse the existing one?